util/customTransformer.js

```diff
@@ -77,7 +78,14 @@ async function runUserTransform(events, code, eventsMetadata, versionId) {
 
   jail.setSync(
     "_log",
-    new ivm.Reference(() => {
+    new ivm.Reference((...args) => {
```

Comment:
Pass testMode into this and enable logs only when testMode = true

Reply:
done